Frameworks/UIKit/StarboardXaml/LayerCoordinator.cpp

```diff
@@ -1060,11 +1061,13 @@ void LayerCoordinator::_VisualHeightChangedCallback(DependencyObject^ sender, De
 
 // ContentGravity
 ContentGravity LayerCoordinator::GetContentGravity(FrameworkElement^ element) {
-    return static_cast<ContentGravity>(element->GetValue(s_contentGravityProperty));
+    // We store as an int (rather than as a ContentGravity enum value) so we can read the values in the VS debugger at runtime
+    return static_cast<ContentGravity>(static_cast<int>(element->GetValue(s_contentGravityProperty)));
```

Comment:
I think the difficulty in debugging this without the change is worth the extra boxing cost (I didn't notice any perf issues with the change).  If this ends up being a hot path, we can revisit though.